src/snyk/advisor/services/AdvisorService.ts

```diff
@@ -0,0 +1,49 @@
+import { AxiosResponse } from 'axios';
+import { Subject } from 'rxjs';
+import { CliError } from '../../cli/services/cliService';
+import { ISnykApiClient } from '../../common/api/apiСlient';
+import { OssResult, OssResultBody, OssVulnerability } from '../../snykOss/ossResult';
+import { ModuleVulnerabilityCount } from '../../snykOss/services/vulnerabilityCount/importedModule';
+import { AdvisorScore } from '../AdvisorTypes';
+
+export default class AdvisorService {
+  protected scores: AdvisorScore[];
+  readonly scanFinished$ = new Subject<void>();
+  private _vulnerabilities: ModuleVulnerabilityCount[];
+  private readonly api = `/unstable/advisor/scores/npm-package`;
+
+  get vulnerabilities(): ModuleVulnerabilityCount[] {
+    return this._vulnerabilities;
+  }
+  private set vulnerabilities(vulnerabilities: ModuleVulnerabilityCount[]) {
```

Comment:
There's no need to work with vulnerability count, since advisor info doesn't depend on that. You probably need to work with `ImportedModule` abstraction that PackageJsonParser gives to you.

Reply:
We need it because the requirement is to show advisor score to packages that have vulnerabilities.